src/appkit.cs

```diff
@@ -11160,6 +11403,17 @@ interface NSPressGestureRecognizer {
 		nint NumberOfTouchesRequired { get; set; }
 	}
 
+	[Protocol]
```

Comment:
availability ?

Reply:
Again, nothing in the header :(

Reply:
10.14 -> https://developer.apple.com/documentation/appkit/NSPasteboardTypeOwner?language=objc